stable/mongodb/0.4.7/chart/values.yaml

```diff
@@ -0,0 +1,74 @@
+_config:
+  _metadata:
+    name: "mongodb"
+    version: "0.4.7"
+    description: ""
+  mongodb:
+    containers:
+      mongodb:
+        image: "bitnami/mongodb:3.4.2-r0"
```

Comment:
These charts all use image from docker hub and can be modified by user before they are deployed.

We can use a private docker registry in next release.